File: Fundamentos de Programacion/C/1. Basicos/1. Secuencia/Ejercicio2.c
/*
2) Calcular el sueldo de un operario ingresando por teclado la cantidad de horas que trabajó en el mes y el valor de la hora. Mostrarle el resultado al usuario con un mensaje adecuado.
*/

#include <stdio.h>

int main() {
    float horasTrabajadas, valorHora, sueldo;

    printf("Ingrese la cantidad de horas trabajadas en el mes: ");
    scanf("%f", &horasTrabajadas);

    printf("Ingrese el valor de la hora: ");
    scanf("%f", &valorHora);

    sueldo = horasTrabajadas * valorHora;

    printf("El sueldo del operario es: %.2f \n", sueldo);

    return 0;
}

File: Algoritmos y Estructura de Datos/Java/src/2. Memoria y Punteros/A. Ejercicios nivel basicos/Ejercicio2.java
class Entero {
    
    int valor;
}

public class Ejercicio2 {
    
    public static void main(String[] args) {

        Entero entero = new Entero();
        entero.valor = 5;

        System.out.println("La referencia del entero es: " + entero);
        System.out.println("El valor del entero es: " + entero.valor);

    }
}
